p/term.ks

```diff
@@ -4,6 +4,7 @@
     clearscreen.
     set terminal:width to w.
     set terminal:height to h.
-    core:doAction("open terminal", true).
+    if career():candoactions
```

Comment:
If we do not do this, and attempt to run term when we have not yet
got to the career point that allows actions, we would crash.